translations/english/repeats.ts

```diff
@@ -3,18 +3,22 @@ import type {Translations} from '../../server/chat';
 export const translations: Translations = {
 	strings: {
 		"Repeated phrases in ${room.title}": "",
-		"No such room: \"${roomid}\".": "",
 		"There are no repeated phrases in ${room.title}.": "",
+		"Action": "",
 		"Phrase": "",
+		"Identifier": "",
 		"Interval": "",
 		"every ${minutes} minute(s)": "",
+		"Raw text": "",
 		"Remove": "",
 		"Remove all repeats": "",
-		"You must specify a numerical interval of at least 1 minute.": "",
-		"The phrase \"${message}\" is already being repeated in this room.": "",
-		"${user.name} set the phrase \"${message}\" to be repeated every ${interval} minute(s).": "",
-		"The phrase \"${target}\" is not being repeated in this room.": "",
-		"${user.name} removed the repeated phrase \"${target}\".": "",
+		"You must specify an interval as a number of minutes between 1 and 1440.": "",
+		"The phrase labeled with \"${id}\" is already being repeated in this room.": "",
+		"${user.name} set the phrase labeled with \"${id}\" to be repeated every ${interval} minute(s).": "",
+		"${user.name} set the Room FAQ \"${topic}\" to be repeated every ${interval} minute(s).": "",
+		"The phrase labeled with \"${id}\" is not being repeated in this room.": "",
+		"The text for the Room FAQ \"${topic}\" is already being repeated.": "",
+		"${user.name} removed the repeated phrase labeled with \"${id}\".": "",
```

Comment:
```suggestion
		"The phrase labeled with \"${id}\" is already being repeated in this room.": "",
		"${user.name} set the phrase labeled with \"${id}\" to be repeated every ${interval} minute(s).": "",
		'${user.name} set the Room FAQ "${topic}" to be repeated every ${interval} minute(s).': "",
		'The phrase labeled with "${id}" is not being repeated in this room.': "",
		'The text for the Room FAQ "${topic}" is already being repeated.': "",
		'${user.name} removed the repeated phrase labeled with "${id}".': "",
```

Reply:
This is TypeScript now, not JSON :P

Reply:
fair, i was just being consistent with all of the other files

Reply:
which other files? I should make a commit to fix them

Reply:
all